src/main/java/org/folio/notes/service/impl/NoteTypesServiceImpl.java

```diff
@@ -0,0 +1,97 @@
+package org.folio.notes.service.impl;
+
+import java.util.UUID;
+import javax.persistence.EntityNotFoundException;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import org.folio.notes.domain.dto.NoteType;
+import org.folio.notes.domain.dto.NoteTypeCollection;
+import org.folio.notes.domain.entity.NoteTypeEntity;
+import org.folio.notes.domain.repository.NoteTypesRepository;
+import org.folio.notes.exception.NoteTypesLimitReached;
+import org.folio.notes.mapper.NoteTypesMapper;
+import org.folio.notes.service.ConfigurationService;
+import org.folio.notes.service.NoteTypesService;
+import org.folio.spring.data.OffsetRequest;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class NoteTypesServiceImpl implements NoteTypesService {
+
+  @Value("${folio.notes.types.default.name}")
+  private String defaultNoteTypeName;
+
+  @Value("${folio.notes.types.default.limit}")
+  private String defaultNoteTypeLimit;
+
+  private static final String NOTE_TYPE_LIMIT_CONFIG = "note-type-limit";
```

Comment:
Please sort your fields in the order:

1. static final
2. static
3. final
4. general